hack/mesh.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# This script can be used to install ServiceMesh on a configured cluster. 
+#
+# This script will:
+#  * Install ServiceMesh
+#
+
+# shellcheck disable=SC1091,SC1090
+source "$(dirname "${BASH_SOURCE[0]}")/lib/__sources__.bash"
+
+set -Eeuo pipefail
+
+exitcode=0
+
+if [[ $UNINSTALL_MESH == "true" ]]; then
+  (( !exitcode )) && uninstall_mesh || exitcode=1
```

Comment:
As there's no additional step before this or, between this and the `exit` , I think we can call just `uninstall_mesh || exit 1`, and a few lines below we can call just `install_mesh || exit 2`. But that's just a nit.

Reply:
Sure, done.